src/main/java/seedu/address/ui/MainWindow.java

```diff
@@ -175,6 +177,44 @@ private void handleSwitchView(VisualizeList list) {
         }
     }
 
+    /**
+     * Scrolls the target pane up
+     *
+     * @param pane
+     */
+    private void scrollUp(String pane) {
+        switch(pane) {
+        case "resultDisplay":
+            resultDisplay.scrollUp();
+            break;
+        case "tabPane":
+            eventListPanel.scrollUp();
+            taskListPanel.scrollUp();
+            reminderListPanel.scrollUp();
+            break;
+        default:
+        }
+    }
+
+    /**
+     * Scrolls the target pane down
+     *
+     * @param pane
+     */
+    private void scrollDown(String pane) {
+        switch(pane) {
+        case "resultDisplay":
+            resultDisplay.scrollDown();
+            break;
+        case "tabPane":
+            eventListPanel.scrollDown();
+            taskListPanel.scrollDown();
+            reminderListPanel.scrollDown();
```

Comment:
This does not seem to be efficient as it is scrolling for all the panels. Is there a way that we can see the panel that is currently being displayed?

Reply:
I will do this in the next iteration.